### DISCUSSION TITLE
Outdoor kitchen design tool

### DISCUSSION THREAD

**Discussion by ConfidantCommunications** [google/model-viewer#4711] [Show and tell]
Hi all,
I've been doing this project for a few years now and over a year ago we added AR with the help of Model Viewer. I'm using Haxe, OpenFL, and Away3D.
<img width="522" alt="image" src="https://github.com/google/model-viewer/assets/1410814/f96e5dc0-58c3-404e-b9d9-708c6992099f">

Design Tool: https://design.rtaoutdoorliving.com/
AR Viewer: https://ar.rtaoutdoorliving.com/ar/2024-01-31_16-47-43_WmG1vPgm679UFvDZeOfw

**Comment by elalish** [comment]
Thanks for sharing! Feel free to @modelviewer on twitter and we'll repost.
